1.Exploratory_Data_Analysis/scripts/0.nomic_cleanup_for_correlation.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import pathlib
+
+import numpy as np
+import pandas as pd
+from sklearn.preprocessing import MinMaxScaler
+
+# In[7]:
+
+
+# Parameters
+cell_type = "SHSY5Y"
+
+
+# In[8]:
+
+
+# set import data paths
+nomic_df_path = pathlib.Path(
+    f"../2.Nomic_nELISA_Analysis/Data/clean/Plate2/nELISA_plate_430420_{cell_type}.csv"
+)
+
+nomic_df_filtered_out_path = pathlib.Path(
+    f"../2.Nomic_nELISA_Analysis/Data/clean/Plate2/nELISA_plate_430420_{cell_type}_cleanup4correlation.csv"
+)
+
+
+# In[9]:
+
+
+# read in data
+nomic_df_raw = pd.read_csv(nomic_df_path)
+
+
+# In[10]:
+
+
+# get the dimensions of the df
+print(f"{nomic_df_raw.shape}_before_filtering")
+# remove column if colname has pgml in it as we are using the Normalised values ('NSU') columns
+nomic_df = nomic_df_raw.loc[
+    :, ~nomic_df_raw.columns.str.contains("pgml", case=False, na=False)
+]
```

Comment:
Would it make sense to show the targeted removal columns instead of the shape? I'm not sure how many there may be so this might be an unreasonable comment, and feel free to ignore if so.

Reply:
Should be 187 less, I added a comment to add to this